contrib/automation_tests/test_cases/connection_window.py

```diff
@@ -135,11 +135,13 @@ def _execute(self):
 
         # Wait for the first data item in the instance list to exist
         # We're not using find_control here because magic lookup enables us to easily wait for the existence of a row
```

Comment:
I guess that comment should be updated

Reply:
Thanks for spotting this, I've updated the comment